www/include/configuration/configObject/contact/formContact.php

```diff
@@ -72,9 +72,14 @@
  * Check if this server is a Remote Server to hide some part of form
  */
 $DBRESULT = $pearDB->query("SELECT i.value FROM informations i WHERE i.key = 'isRemote'");
-$isRemote = array_map("myDecode", $DBRESULT->fetchRow());
-$DBRESULT->closeCursor();
-$isRemote = ($isRemote['value'] === 'yes') ? true : false;
+$result = $DBRESULT->fetchRow();
+if ($result === false) {
+    $isRemote = false;
+} else {
+    $isRemote = array_map("myDecode", $DBRESULT->fetchRow());
```

Comment:
$DBRESULT->fetchRow() will always return false, isn't it ?
Cause it is already called previously and I guess there is only one value for isRemote

Reply:
I can't see where it is called earlier, my idea was just to make additional check.

Reply:
Fixed